source/clear-linux/guides/guides.rst

```diff
@@ -0,0 +1,13 @@
+.. _guides:
+
+Guides
+######
+
+The following guides provide step by step instruction for all tasks that come

```

Comment:
Consider changing "step by step instruction for all tasks that come" to " step-by-step instructions for tasks"